Failed to pass the run_tests.py.

Hi,

I install the git master version of seekpath will pip's editable mode with the following steps:

```
# Doing this within pyenv using python 3.8.1:
$ pyenv shell 3.8.1
# Create a virtualenv named as hpc for this test:
$ pyenv virtualenv hpc
$ pyenv shell hpc 
$ git clone https://github.com/giovannipizzi/seekpath.git seekpath.git
$ cd seekpath.git
$ pip install -e .
```

And I also installed the all of the packages listed in the *requirements.txt files.  
Then I do the following testing:

```
$ python run_tests.py 
ssF.ssss..................................................................
======================================================================
FAIL: test_2 (seekpath.brillouinzone.test_brillouinzone.TestBZ)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/werner/Public/hpc/tools/seekpath.git/seekpath/brillouinzone/test_brillouinzone.py", line 254, in test_2
    self.assertTrue(is_same,
AssertionError: False is not true : The two sets of faces are different: The following item in the first list was not found in the second one: [[-0.5, 0.5, -0.3400000000000001], [0.30000000000000004, 0.5, -0.4999999999999999], [0.3, 0.3, -0.6599999999999999], [-0.5, 0.3, -0.4999999999999999]]

----------------------------------------------------------------------
Ran 74 tests in 0.340s

FAILED (failures=1, skipped=6)
```

Any hints for this issue?

Regards

--- a/seekpath/__init__.py
+++ b/seekpath/__init__.py
@@ -7,7 +7,7 @@ Author: Giovanni Pizzi, EPFL (2016-2020)
 Licence: MIT License, see LICENSE.txt file
 """
 
-__version__ = "1.9.4"
+__version__ = "1.9.5"
 __author__ = "Giovanni Pizzi, EPFL"
 __copyright__ = "Copyright (c), 2016-2020, Giovanni Pizzi, EPFL (Theory and Simulation of Materials (THEOS) and National Centre for Computational Design and Discovery of Novel Materials (NCCR MARVEL)), Switzerland."
 __credits__ = ["Yoyo Hinuma"]

--- a/seekpath/brillouinzone/test_brillouinzone.py
+++ b/seekpath/brillouinzone/test_brillouinzone.py
@@ -59,7 +59,7 @@ def is_same_face(f1, f2):
     if found:
         return True
     # Try also reversed order
-    indices = (-(np.arange(len(f2)) + shift)) % len(f2)
+    indices = (-np.arange(len(f2)) + shift) % len(f2)
     err = np.abs(np.array(f1) - np.array(f2)[indices]).mean()
     found = err < threshold
     return found

